packages/stylelint-plugin/src/rules/no-invalid-use-of-var-tokens-in-equations.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { getParser } from "../util/utils"
+import { noInvalidUseOfVarTokensInEquations } from "./no-invalid-use-of-var-tokens-in-equations"
+
+describe("no-invalid-use-of-var-tokens-in-equations rule", () => {
+  test.each([
+    ["$kz-color-wisteria-800 * 5", true],
+
+    ["$kz-var-spacing-md 5px - 2px", false],
+
+    ["$kz-something-user-defined + 2", false],
+
+    ["solid 1px + $kz-spacing-md black", true],
+
+    ["calc($kz-spacing-md + 5px)", true],
+
+    ["calc($kz-var-spacing-md + 5px)", true],
+
+    ["calc(#{$kz-var-spacing-md} + 5rem)", false],
+    ["-$kz-var-spacing-md", true],
```

Comment:
It might be handly to add a test case for adjacent operators, something like these:
```
["calc(#{$kz-var-spacing-md} * -5rem)", false],
["calc($kz-spacing-md * -5px)", true],
```

Reply:
I've added these test cases, and refactored to a more traditional `test()` structure, rather than `test.each()` so I can clarify what each one is testing, because it wasn't obvious to me